### PR TITLE
feat(atom/panel): added border-radius

### PR DESCRIPTION
**Pull request by ferransimon**

### CODE REVIEW

File: components/atom/panel/src/index.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 @import '~@s-ui/theme/lib/index';
 
+$bdrs-atom-panel: 0px !default;
```
Comment (midudev) [inline components/atom/panel/src/index.scss]: Es raro que no se queje, pero diría que los valores que son 0 no van con unidades.

En lugar de `0px` debería ser `0`.

**Review by midudev — APPROVED** [review]

**Review by nucliweb — APPROVED** [review]

**Review by javiauso — APPROVED** [review]

File: components/atom/panel/src/index.scss [hunk 2]
```diff
@@ -32,6 +33,7 @@ $vertical-alignments-atom-panel: ('top', 'center', 'bottom') !default;
   }
 
   &-color {
+    border-radius: $bdrs-atom-panel;
```
Comment (PablitoGS) [inline components/atom/panel/src/index.scss]: Creo que no deberiamos asociar el border radius al panel color, en todo caso, en la vista mobile el panel-color no tiene border radius en IJ

File: components/atom/panel/src/index.scss [hunk 3]
```diff
@@ -1,5 +1,6 @@
 @import '~@s-ui/theme/lib/index';
 
+$bdrs-atom-panel: 0 !default;
```
Comment (andresin87) [inline components/atom/panel/src/index.scss]: `$bdrs-atom-panel: $bdrs-none !default`

**Review by andresin87 — CHANGES_REQUESTED** [review]

File: components/atom/panel/src/index.scss (hunk 3 above)
Comment (andresin87) [reply components/atom/panel/src/index.scss]: @s-ui/theme/lib/settings/_box-style.scss

File: components/atom/panel/src/index.scss (hunk 2 above)
Comment (turolopezsanabria) [reply components/atom/panel/src/index.scss]: If we think mobile first, the `$bdrs` should be `none` by default, and apply a media query to be rounded only for Desktop.

Comment (andresin87) [reply components/atom/panel/src/index.scss]: Both Components, ColorPanel && ImagePannel must include its baseClassName = `sui-atom-panel`

File: components/atom/panel/src/index.scss (hunk 3 above)
Comment (ferransimon) [reply components/atom/panel/src/index.scss]: I moved that to a component prop so now I left the default value to `$bdrs-l` so in the demo we can see a difference between a floating panel or a non-floating panel

File: components/atom/panel/src/index.scss (hunk 2 above)
Comment (ferransimon) [reply components/atom/panel/src/index.scss]: They do not have this baseClassName, there is a baseClassName for every single Panel (color, image) if we need to do a refactor on this maybe it gets out of the scope of this PR

**Review by andresin87 — APPROVED** [review]

**Pull request merged**